osu.Server.Spectator/Hubs/MultiplayerHubContext.cs

```diff
@@ -166,15 +167,15 @@ public async Task StartMatch(ServerMultiplayerRoom room)
             if (room.Queue.CurrentItem.Expired)
                 throw new InvalidStateException("Cannot start an expired playlist item.");
 
-            var readyUsers = room.Users.Where(u => u.State == MultiplayerUserState.Ready).ToArray();
-
             // If no users are ready, skip the current item in the queue.
-            if (readyUsers.Length == 0)
+            if (!room.Users.Any(u => u.State == MultiplayerUserState.Ready))
             {
                 await room.Queue.FinishCurrentItem();
```

Comment:
```suggestion
```
Also, according to `MultiplayerMatchStartCountdownTest.AutoStartCountdownContinuesWhileAllUsersNotReady` test we want to skip current item if no one is ready.

I think it would make more sense to keep item in the queue after countdown because no one was ready for the item to begin. After countdown, host can either restart countdown or remove the item manually if it is not desired. Perhaps this change would be better suited for a separate PR?